docs/hoon/library/2b.md

```diff
@@ -4,28 +4,132 @@ navhome: /docs/
 
 
 # 2b: list logic
-Reverse
+### `++fand`
+
+All indices in `list`
+
+Produces the indices of all occurrences of `list` `nedl` in
+`list` `hstk` as a `list` of atoms.
+
```

Comment:
The "All indices in `list`" part looks out of place, and redundant with the fuller explanation just below.

Also, I'm pretty sure this next complaint is my fault not yours---but the `list` `nedl` and `list` `hstk` don't come out looking right.  I think we should just drop the `list` in each case.  We define `nedl` and `hstk` as lists immediately below, so this shouldn't cause any confusion.